<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);
border-radius: 3px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="TitleBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
image: url(:/images/icons/nogo-01.png);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>Nhập ID bài học</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="btn_frame">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_minimize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background: transparent;
	border-radius: 15px;
}
QPushButton:hover {
	background: rgb(108, 108, 108);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/minimize_white.png</normaloff>:/images/icons/minimize_white.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_quit">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background: transparent;
	border-radius: 15px;
}
QPushButton:hover {
	background: rgb(255, 0, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/close_white.png</normaloff>:/images/icons/close_white.png</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);</string>
      </property>
      <property name="text">
       <string>Tải xuống đã hoàn tất.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="id_entry">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>iCiel Effra Regular</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nhập mã ID vào đây</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="In_btn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
         </property>
         <property name="locale">
          <locale language="Vietnamese" country="Vietnam"/>
         </property>
         <property name="text">
          <string>Truy cập phòng</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="room_btn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
         </property>
         <property name="locale">
          <locale language="Vietnamese" country="Vietnam"/>
         </property>
         <property name="text">
          <string>Tạo ID phòng</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Go_Room">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
         </property>
         <property name="locale">
          <locale language="Vietnamese" country="Vietnam"/>
         </property>
         <property name="text">
          <string>Vào phòng</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Quit">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
         </property>
         <property name="locale">
          <locale language="Vietnamese" country="Vietnam"/>
         </property>
         <property name="text">
          <string>Rời phòng</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>Go_Room</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>Quit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>id_entry</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>In_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>room_btn</sender>
   <signal>clicked()</signal>
   <receiver>Go_Room</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>room_btn</sender>
   <signal>clicked()</signal>
   <receiver>Quit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
